internal/cmd/resolve/resolve.go

```diff
@@ -79,6 +81,15 @@ $ debricked resolve . `+exampleFlags)
 
 	cmd.Flags().BoolP(NpmPreferredFlag, "", npmPreferred, npmPreferredDoc)
 
+	cmd.Flags().IntVarP(&resolutionStrictness, ResolutionStrictFlag, "s", file.StrictAll, `Allows you to configure exit code 1 or 0 depending on if the resolution was successful or not.
```

Comment:
I'm not sure we should use the shortcut "s" here, will this flag be used often enough to require this?